pubnub-kotlin/pubnub-kotlin-api/src/jsMain/kotlin/com/pubnub/kmp/converters.js.kt

```diff
@@ -26,33 +27,33 @@ internal fun SetChannelMetadataResponse.toChannelMetadataResult(): PNChannelMeta
 internal fun PubNub.ChannelMetadataObject.toChannelMetadata(): PNChannelMetadata {
     return PNChannelMetadata(
         id,
-        name,
-        description,
-        custom?.toMap(),
-        updated,
-        eTag,
-        type,
-        status
+        PatchValue.of(name), // TODO support optionals
+        PatchValue.of(description),
+        PatchValue.of(custom?.toMap()),
+        PatchValue.of(updated),
+        PatchValue.of(eTag),
+        PatchValue.of(type),
+        PatchValue.of(status),
     )
 }
 
 internal fun PubNub.UUIDMembershipObject.toPNMember() = PNMember(
     PNUUIDMetadata(
         uuid.id,
-        uuid.name,
-        uuid.externalId,
-        uuid.profileUrl,
-        uuid.email,
-        uuid.custom?.toMap(),
-        uuid.updated,
-        uuid.eTag,
-        uuid.type,
-        uuid.status
+        PatchValue.of(uuid.name), // TODO support optionals
```

Comment:
What does it mean "support optionals" here?

Reply:
right now it just takes nulls and puts them in PatchValue so it's not really optional, it's just always a value (which can be null)

I have to see if there's a way to get the information from JS SDK about the presence (or absence) of value